sdk/src/main/scala/com/akkaserverless/javasdk/impl/valueentity/AnnotationBasedEntitySupport.scala

```diff
@@ -64,18 +64,20 @@ private[impl] class AnnotationBasedEntitySupport(
       })
     }
 
-    override def handleCommand(command: JavaPbAny, context: CommandContext[JavaPbAny]): Reply[JavaPbAny] = unwrap {
-      behavior.commandHandlers.get(context.commandName()).map { handler =>
-        val adaptedContext =
-          new AdaptedCommandContext(context, anySupport)
-        handler.invoke(entity, command, adaptedContext)
-      } getOrElse {
-        throw EntityException(
-          context,
-          s"No command handler found for command [${context.commandName()}] on $behaviorsString"
-        )
-      }
-    }
+    override def handleCommand(command: JavaPbAny, context: CommandContext[JavaPbAny]): ValueEntityEffect[JavaPbAny] =
+      ??? // FIXME this will be removed
```

Comment:
I changed the signature to return ValueEntityEffect instead of Reply. Not sure if we are ready for that change yet?

Reply:
it's probably best to revert this and related changes in this PR, so that we can have green tests some longer before switching over

Reply:
yeah, tricky - would be nice to glue it together, but as it's going away eventually anyway we should be careful not to put *too* much energy in the glue... if it's possible to keep it out of this PR and perhaps do the next step in a next PR that might be convenient